### DISCUSSION TITLE
Dropdown组件的mouseenter事件如何通过jquery自动触发mouseenter呢

### DISCUSSION THREAD

**Discussion by bihongbin3027** [ant-design/ant-design#47169] [Q&A] [answered]
我使用$('.ant-dropdown-trigger').trigger('mouseenter') 无法触发hover

**Comment by zombieJ** [comment, marked as answer]
感觉是个 XY 问题，如果你是需要手工触发下拉效果，应该通过受控属性 `open` 才对。

**Reply by bihongbin3027** [reply]
我做了一个浏览器插件 需要用jquery找到这个元素 然后自动触发hover打开下拉菜单 
![image](https://github.com/ant-design/ant-design/assets/15045200/562b5707-c0e5-48fc-a432-2286a7d7e036)
